content/kubermatic/master/tutorials_howtos/_index.en.md

```diff
@@ -5,8 +5,12 @@ weight = 20
 +++
 
 
-This section contains tutorials that will show you how to execute basic tasks with your KKP cluster.
+Are you just embarking on your cloud adoption journey but feeling a bit overwhelmed on how to get started? Then you’re in the right place here:)
```

Comment:
I am not sure, but is it really a good idea to have emojis in the documentation?

Reply:
Removed it.